de.json

```
@@ -618,19 +618,19 @@
         "Credits.Send" : "Senden",
         "Credits.Sending" : "Sende...",
         "Credits.Success" : "Erfolgreich!\nSie können dieses Fenster schließen.",
-        "Credits.BlockchainWarning" : "Hinweis: der Transfer zur Blockchain kann bis zu 48 Stunden dauern.",
+        "Credits.BlockchainWarning" : "Hinweis: Der Transfer zur Blockchain kann bis zu 48 Stunden dauern.",
 
         "NewWorld.Title" : "Neue Welt",
         "NewWorld.Template" : "Weltvorlage:",
-        "NewWorld.CustomPreset" : "Nutzerdefinierte Voreinstellung:",
+        "NewWorld.CustomPreset" : "Benutzerdefinierte Voreinstellung:",
         "NewWorld.SessionSettings" : "Sitzungs-Einstellungen:",
         "NewWorld.UnsafeMode" : "Unsicherer Modus:",
         "NewWorld.Port" : "Port:",
         "NewWorld.AutoPort" : "Auto:",
         "NewWorld.DefaultName" : "{name} Welt",
 
         "FullBody.Title" : "Ganzkörper-Kalibrierung",
-        "FullBody.SetHeight" : "1) Setzen Sie Ihre Größe",
+        "FullBody.SetHeight" : "1) Geben Sie Ihre Größe ein",
```

```suggestion
        "FullBody.SetHeight" : "1) Geben Sie Ihre Höhe ein",
```
Why don't we use the word for height instead of size?

---

Both are interchangeable, Wikipedia uses "Größe" (and Körperhöhe redirects there to Körpergröße again) so we can go with the flow.